src/Resources/contao/modules/ModuleLogin.php

```diff
@@ -33,12 +33,6 @@ class ModuleLogin extends \Module
 	 */
 	protected $strTemplate = 'mod_login';
 
-	/**
-	 * Flash type
-	 * @var string
-	 */
-	protected $strFlashType = 'contao.FE.error';
```

Comment:
Please do not remove this but reset it to `protected $strFlashType = 'contao.' . TL_MODE . '.error';` like it is in Contao 4.4.

Reply:
Fixed in 6ad2a9ce87ea617a5e78ce463e5049df81008cff